lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -329,6 +344,7 @@ private void repaint() {
 
             repaintPosition(scrollPane.hvalueProperty().doubleValue());
         }
+        getMiniMapController().drawMiniMap();
```

Comment:
We've been using shouts for inter controller communication. Why not use it here?